src/app/pages/user/dialogs/update/update-user.dialog.html

```diff
@@ -56,17 +56,16 @@ <h3 mat-dialog-title>עדכון פרטים אישיים</h3>
           <div class="msw-form-row">
             <mat-form-field color="primary">
               <mat-select placeholder="כיתה" >
-                <mat-option [value]="">בחר</mat-option>            
+                <mat-option [value]="">בחר</mat-option>
               </mat-select>
             </mat-form-field>
           </div>
         </div>
-      </form>
-    </mat-dialog-content>
-    <div mat-dialog-actions>
-      <button  mat-button class="msw-button-cancel" tabindex="-1" (click)="close()">ביטול</button>
-      <!-- <button [type]="submit" class="mat-raised-button mat-primary" (click)="updateDetails()" [disabled]="!formControl.valid" >עדכן משתמש</button> -->
-      <button  mat-button class="msw-button-update" (click)="updateDetails()" [disabled]="!formControl.valid" >עדכן</button>
-    </div>
+      </mat-dialog-content>
+      <div mat-dialog-actions>
+        <button  mat-button class="msw-button-cancel" tabindex="-1" (click)="close()">ביטול</button>
+         <!-- <button [type]="submit" class="mat-raised-button mat-primary" (click)="updateDetails()" [disabled]="!formControl.valid" >עדכן משתמש</button> -->
```

Comment:
I see this comment was before but if you fixed the issue around it, can you please remove the comment as well?

Thanks.

Reply:
Sure thing.
Done